docs/books/web_services/053-load-balancer-proxies-squid.md

```diff
@@ -0,0 +1,458 @@
+---
+author: Antoine Le Morvan
+contributors: 
+title: Part 5.3 Squid
+tags:
+  - squid
+  - proxy
+  - http
+---
+
+## Squid
+
+In this chapter, you will learn about Squid, the HTTP proxy cache.
+
+****
+
+**Objectives**: In this chapter, you will learn how to:
+
+:heavy_check_mark: install squid  
+:heavy_check_mark: configure it to proxify and cache http content.  
+
+:checkered_flag: **squid**, **proxy**, **http**
+
+**Knowledge**: :star: :star:  
+**Complexity**: :star: :star:  
+
+**Reading time**: 20 minutes
+
+****
+
+### Generalities
+
+Setting up a proxy server involves choosing between two types of architecture:
+
+* A standard proxy architecture, requiring specific configuration of each client and their web browsers,
+* Captive proxy architecture, which involves intercepting the frames sent by the client and rewriting them to the proxy server.
+
+In either case, there is a break in the network: A client can no longer physically address a remote server directly, without going through a proxy server.
+
+The client workstation is protected by two firewalls and never communicates directly with the outside network.
+
+![Proxy-based architecture](img/squid-architecture-proxy.png)
+
+!!! NOTE
+
+    This architecture requires browser configuration on the client workstation.
+
+With a captive proxy, there's no need to configure all client workstations.
```

Comment:
```suggestion
With a captive proxy, there is no need to configure all client workstations.
```

Reply:
see the section **Avoid Contractions** in the documentation:
https://docs.rockylinux.org/guides/contribute/style_guide/#voice-and-tone